scripts/Diffraction/isis_powder/polaris_routines/polaris_algs.py

```diff
@@ -178,13 +179,13 @@ def _load_qlims(q_lims):
 
 def _determine_chopper_mode(ws):
     if ws.getRun().hasProperty('Frequency'):
-        frequency = ws.getRun()['Frequency'].lastValue()
-        print("No chopper mode provided")
-        if frequency == 50:
-            print("automatically chose Rietveld")
+        frequency = ws.getRun()['Frequency'].timeAverageValue()
+        print("Found chopper frequency of {} in log file.".format(frequency))
+        if math.isclose(frequency, 50, rel_tol=1):
+            print("Automatically chose Rietveld mode")
             return 'Rietveld', polaris_advanced_config.rietveld_focused_cropping_values
-        if frequency == 0:
-            print("automatically chose PDF")
+        if math.isclose(frequency, 0, rel_tol=1):
```

Comment:
I think in this you want to have `abs_tol`, using `rel_tol` means that any value from 0-100 will be counted as Rietveld, and there are no values that are conted as PDF.

Reply:
Thanks @Harrietbrown good spot. Clearly I didn't have my head on straight this morning!